releasing.md

```diff
@@ -0,0 +1,11 @@
+# Releasing mjolnir
+
+1. Create a new branch and edit the `version` variable of `package.json` to reflect the new version
+2. Once that branch has been merged, switch back to the main branch and pull in the new changes
+3. Tag the new version, ie `git tag -s vX.Y.Z` (where vX.Y.Z is the new version), and push the tag
```

Comment:
@H-Shay There should be a Mjolnir wiki within the Element wiki (if such a thing still exists), I'm not sure why these were never transferred to this repository but there were instructions for how to release Mjolnir there that are still accurate. Mjolnir was released with the `yarn version` script to ensure that the `version` in package.json, the version within the synapse module, and the git tag are all synchronised without causing a mistake from typos (this has happened previously). There were also instructions for how to push the release commit and the tag atomically so that someone new wouldn't push only one or the other.

Reply:
Thanks for the insight! I've found the entry.